canonical-web-and-design/canonical.com
# Fix /careers/results

## Done

- on /careers/results, disable checkboxes until we're able to handle multiple job applications
- includes a lot of groundwork for /careers/apply that we can use when the above is implemented (WIP in this branch [here](https://github.com/sowasred2012/canonical.com/tree/results-apply-mvp))

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers/start
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Select your options and submit them, see that you are taken to /careers/results, and that the items in the list _don't_ feature checkboxes.


## Issue / Card

Fixes #129
